### DISCUSSION TITLE
Android App doesn‘t update at all

### DISCUSSION THREAD

**Discussion by SiggiKeller** [owntracks/android#1821] [Q&A]
Hello,

On my Samsung S22+ with Android 14 I installed OwnTracks. I can update the location manually - it‘s written to the server. But when I put it in some of the other modes the location is never updated. So I leave home for a walk and all the time my home address is shown in the status of OwnTracks until I manually open the app.
It looks like the app is put on hold (sleep) whenever it‘s in background. Of course I made all necessary changes to the battery saving settings of this app.
Any idea?

Thanks,
Siggi

**Comment by jpmens** [comment]
I think this is a duplicate of https://github.com/owntracks/android/issues/1759
